doc/content/use-cases/packet-injector.md

```diff
@@ -12,25 +14,44 @@ To use the packet injector we need to provide the below parameters,
 * Source MAC of the packet, optional if source node given.
 * Destination IP of the packet, optional if destination node given.
 * Destination MAC of the packet, optional if destination node given.
-* Type of packet. currently only ICMP is supported.
+* Type of packet. currently ICMP, TCP and UDP are supported.
 * Number of packets to be generated, default is 1.
+* IMCP ID, used only for ICMP type packets.
+* Interval, the delay between two packets in milliseconds.
+* Payload
+* Source port, used only for TCP and UDP packets, if not given generates one randomly.
+* Destination port, used only for TCP and UDP packets, if not given generates one randomly.
 
 ```console
-$ skydive client inject-packet [flags]
+$ skydive client inject-packet create [flags]
 
 Flags:
       --count int        number of packets to be generated (default 1)
       --dst string       destination node gremlin expression
       --dstIP string     destination node IP
       --dstMAC string    destination node MAC
+      --dstPort int      destination port for TCP packet
+      --id int           ICMP identification
+      --interval int     wait interval milliseconds between sending each packet (default 1000)
       --payload string   payload
-      --src string       source node gremlin expression
+      --src string       source node gremlin expression (mandatory)
       --srcIP string     source node IP
       --srcMAC string    source node MAC
-      --type string      packet type: icmp (default "icmp")
+      --srcPort int      source port for TCP packet
+      --type string      packet type: icmp4, icmp6, tcp4, tcp6, udp4 and udp6 (default "icmp4")
+```
+
+## Example
+```console
+$ skydive client inject-packet create --src="G.V().Has('TID', 'feae10c1-240e-48e0-4a13-c608ffd157ab')" --dst="G.V().Has('TID', 'feae10c1-240e-48e0-4a13-c608ffd15700')" --type="icmp4" --count=15
```

Comment:
Maybe you can show the output of the command. And use the returned UUID for the `inject-packet delete` command below